platform/view/services/storage/driver/sql/common/simplekeydata.go

```diff
@@ -56,7 +56,7 @@ func (db *SimpleKeyDataStore) PutData(ctx context.Context, key string, data []by
 		Row(key, data).
 		OnConflictDoNothing().
 		Format()
-	logger.Debug(query, params)
+	logger.Debug(query, string(data))
```

Comment:
in this case, we don't need the DebugfContext because an event in the trace will be added by the call to ExecContext later.